ExploreASL_Initialize.m

```diff
@@ -697,111 +639,242 @@
+    % Runs following steps:
+    % 1) Set default settings if not defined
+    % 2) Print data/study specific settings
+    % 3) Print warnings
+
+    %% -----------------------------------------------------------------------
+    %% 1) Set default settings if not defined
+    if ~isfield(x,'Quality') || (x.Quality~=0 && x.Quality~=1)
+        x.Quality = 1;
+        fprintf('%s\n', 'Default Quality=1 used (optimal quality)');
+    end
+    if ~isfield(x,'DELETETEMP') || (x.DELETETEMP~=0 && x.DELETETEMP~=1)
+        x.DELETETEMP = 1;
+    %     fprintf('%s\n','Default x.DELETETEMP=1 used (delete files temporarily used for processing)');
+    end
 
-%% -----------------------------------------------------------------------
-%% 2) Print data/study specific settings
-fprintf('%s\n','-------------------------------------------');
-fprintf('%s\n\n','ExploreASL will run with following settings:');
-fprintf('%s\n\n',['Root folder = ' x.D.ROOT]);
-
-if x.nWorkers>1
-    fprintf(['I am worker ' num2str(x.iWorker) '/' num2str(x.nWorkers) '\n']);
-    fprintf('Note that the resulting number of scans mentioned below applies only to this worker\n');
-end
+    %% -----------------------------------------------------------------------
+    %% 2) Print data/study specific settings
+    fprintf('==================================== Additional Settings =====================================\n');
+    
+    if x.nWorkers>1
+        fprintf(['I am worker ' num2str(x.iWorker) '/' num2str(x.nWorkers) '\n']);
+        fprintf('Note that the resulting number of scans mentioned below applies only to this worker\n');
+    end
 
-fprintf('%s\n',[num2str(x.nTotalSubjects) ' scans - ' num2str(x.nExcluded) ' exclusions, resulting in ' num2str(x.nSubjects) ' scans of: ']);
+    fprintf('%s\n',[num2str(x.nTotalSubjects) ' scans - ' num2str(x.nExcluded) ' exclusions, resulting in ' num2str(x.nSubjects) ' scans of: ']);
 
-for iT=1:x.nTimePointsTotal
-    fprintf('%s\n',['Longitudinal timePoint ' num2str(iT) ' = ' num2str(x.nTimePointTotalSubjects(iT)) ' scans - ' num2str(x.nTimePointExcluded(iT)) ' exclusions = ' num2str(x.nTimePointSubjects(iT)) ' scans']);
-end
+    for iT=1:x.nTimePointsTotal
+        fprintf('%s\n',['Longitudinal timePoint ' num2str(iT) ' = ' num2str(x.nTimePointTotalSubjects(iT)) ' scans - ' num2str(x.nTimePointExcluded(iT)) ' exclusions = ' num2str(x.nTimePointSubjects(iT)) ' scans']);
+    end
 
-fprintf('%s\n',['ASL sessions: ' num2str(x.nSessions)]);
+    fprintf('%s\n',['ASL sessions: ' num2str(x.nSessions)]);
 
-fprintf('\n%s\n','Ancillary data, sets:');
-if isfield(x.S,'SetsID')
-        fprintf('%s\n',[num2str(size(x.S.SetsID,2)) ' sets are defined for ' num2str(size(x.S.SetsID,1)) ' "SubjectsSessions":']);
+    fprintf('\n%s','Ancillary data, sets: ');
+    if isfield(x.S,'SetsID')
+            fprintf('%s\n',[num2str(size(x.S.SetsID,2)) ' sets are defined for ' num2str(size(x.S.SetsID,1)) ' "SubjectsSessions"']);
```

Comment:
why did you delete the `:` here?

Reply:
I changed the overall look a little bit. Check out the following example for loading a dataset. If you don't like it, we can change it back.

```
==================================== ExploreASL Settings =====================================
DataParPath		...pment\MATLAB\m.stritt\TestDataSet\analysis\DataParameters_LowQ.json
Run DCM2NII		0
Run NII2BIDS		0
Run ANONYMIZE		0
Run BIDS2LEGACY		0
Run Structural Module	0
Run ASL Module		0
Run Population Module	0
SkipPause		1
iWorker			1
nWorkers		1
==============================================================================================
ExploreASL v1.5.1 initialized ... 
Automatically defining sessions...
==================================== Additional Settings =====================================
1 scans - 0 exclusions, resulting in 1 scans of: 
Longitudinal timePoint 1 = 1 scans - 0 exclusions = 1 scans
ASL sessions: 1

Ancillary data, sets: 3 sets are defined for 1 "SubjectsSessions"
Set 1 = "LongitudinalTimePoint" options "TimePoint_1", codes for paired data
Set 2 = "SubjectNList" options "SubjectNList", codes for paired data
Set 3 = "Site" options "SingleSite", codes for two-sample data

x.D.ROOT		M:\SoftwareDevelopment\MATLAB\m.stritt\TestDataSet\analysis
x.DELETETEMP		1 (delete temporary files)
x.Quality		0 (0 = fast try-out; 1 = normal high quality)

==============================================================================================
```

Reply:
Very nice! I would then only make it a bit more compact, by writing all ImportModules on 1 line, and all ProcessModules on 1 line.

Very cool!

Reply:
Sure, if you prefer that. Maybe like this? d358bb6

```
>> [x] = ExploreASL_Master('.\analysis\DataParameters_LowQ.json', 1, 1);
ExploreASL will run the import workflow and will run the processing pipeline...


==============================================================================================
 ________                      __                                 ______    ______   __        
/        |                    /  |                               /      \  /      \ /  |      
########/  __    __   ______  ## |  ______    ______    ______  /######  |/######  |## |      
## |__    /  \  /  | /      \ ## | /      \  /      \  /      \ ## |__## |## \__##/ ## |      
##    |   ##  \/##/ /######  |## |/######  |/######  |/######  |##    ## |##      \ ## |      
#####/     ##  ##<  ## |  ## |## |## |  ## |## |  ##/ ##    ## |######## | ######  |## |      
## |_____  /####  \ ## |__## |## |## \__## |## |      ########/ ## |  ## |/  \__## |## |_____ 
##       |/##/ ##  |##    ##/ ## |##    ##/ ## |      ##       |## |  ## |##    ##/ ##       |
########/ ##/   ##/ #######/  ##/  ######/  ##/        #######/ ##/   ##/  ######/  ########/ 
                    ## |                                                                      
                    ## |                                                                      
                    ##/  

==================================== ExploreASL Settings =====================================
DataParPath		...pment\MATLAB\m.stritt\TestDataSet\analysis\DataParameters_LowQ.json
Import Modules		DCM2NII NII2BIDS ANONYMIZE BIDS2LEGACY
Process Modules		Structural ASL Population
bPause			0
iWorker			1
nWorkers		1
==============================================================================================
ExploreASL v1.5.1 initialized ... 
```

```
>> [x] = ExploreASL_Master('.\analysis\DataParameters_LowQ.json', 0, 0);
ExploreASL will load the dataset...


==============================================================================================
 ________                      __                                 ______    ______   __        
/        |                    /  |                               /      \  /      \ /  |      
########/  __    __   ______  ## |  ______    ______    ______  /######  |/######  |## |      
## |__    /  \  /  | /      \ ## | /      \  /      \  /      \ ## |__## |## \__##/ ## |      
##    |   ##  \/##/ /######  |## |/######  |/######  |/######  |##    ## |##      \ ## |      
#####/     ##  ##<  ## |  ## |## |## |  ## |## |  ##/ ##    ## |######## | ######  |## |      
## |_____  /####  \ ## |__## |## |## \__## |## |      ########/ ## |  ## |/  \__## |## |_____ 
##       |/##/ ##  |##    ##/ ## |##    ##/ ## |      ##       |## |  ## |##    ##/ ##       |
########/ ##/   ##/ #######/  ##/  ######/  ##/        #######/ ##/   ##/  ######/  ########/ 
                    ## |                                                                      
                    ## |                                                                      
                    ##/  

==================================== ExploreASL Settings =====================================
DataParPath		...pment\MATLAB\m.stritt\TestDataSet\analysis\DataParameters_LowQ.json
Import Modules			   
Process Modules			  
bPause			0
iWorker			1
nWorkers		1
==============================================================================================
ExploreASL v1.5.1 initialized ... 
Automatically defining sessions...
==================================== Additional Settings =====================================
1 scans - 0 exclusions, resulting in 1 scans of: 
Longitudinal timePoint 1 = 1 scans - 0 exclusions = 1 scans
ASL sessions: 1

Ancillary data, sets: 3 sets are defined for 1 "SubjectsSessions"
Set 1 = "LongitudinalTimePoint" options "TimePoint_1", codes for paired data
Set 2 = "SubjectNList" options "SubjectNList", codes for paired data
Set 3 = "Site" options "SingleSite", codes for two-sample data

x.D.ROOT		M:\SoftwareDevelopment\MATLAB\m.stritt\TestDataSet\analysis
x.DELETETEMP		1 (delete temporary files)
x.Quality		0 (0 = fast try-out; 1 = normal high quality)

==============================================================================================
```